### ISSUE TITLE
<amp-carousel> is animating cards while user is scrolling in iOS 13 beta (Safari, Chrome & Firefox)

### ISSUE THREAD

**Issue by sirceljm-celtra** [ampproject/amphtml#24541]
## What's the issue?

If user is scrolling on the AMP page which includes an `<amp-carousel>`, then the carousel will switch between cards while scrolling.

If "autoplay" is set on the carousel then the carousel will be animating indefinitely while scrolling. If "autoplay" is not set, then the carousel will animate back to the first card and stay there.

## How do we reproduce the issue?

Open the following AMP page in any browser in iOS13 beta on an iPhone or iPad. 
```
<!doctype html>
<html ⚡>
<head>
<meta charset="utf-8">
  <title>amp-carousel</title>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
  <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
  <link rel="canonical" href="https://amp.dev/documentation/examples/components/amp-carousel/index.html">
  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
</head>
<body>
  <amp-carousel width="400" height="300" layout="responsive" type="slides" autoplay delay="2000">
    <amp-img src="https://playground.amp.dev/static/samples/img/image1.jpg" width="400" height="300" layout="responsive" alt="a sample image"></amp-img>
    <amp-img src="https://playground.amp.dev/static/samples/img/image2.jpg" width="400" height="300" layout="responsive" alt="another sample image"></amp-img>
    <amp-img src="https://playground.amp.dev/static/samples/img/image3.jpg" width="400" height="300" layout="responsive" alt="and another sample image"></amp-img>
  </amp-carousel>
</body></html>
```

When scrolling the page the carousel will animate between cards very fast.

## What browsers are affected?

All browsers on iOS13 beta 

## Which AMP version is affected?

Latest

**Comment by itsmsc** [comment]
The problem still exists in the iOS 13 - release version. In my opinion is this a serious UI - issue

**Comment by meistudioli** [comment]
I have some problem too.

demo: https://blog.lalacube.com/mei/yahoo_mall_fp_amp.php

![image](https://user-images.githubusercontent.com/10822546/65516129-02736380-df13-11e9-9f7d-4c05b6d50734.png)

**Comment by tanakayuki** [comment]
I'm seeing the same symptoms
Occurs even when “autoplay” is not set in the amp-carousel
, can confirm this bug in iOS(+13)

**Comment by sparhami** [comment]
I created a demo page to reproduce the issue with amp-carousel 0.1:

https://output.jsbin.com/qujeqah/1

Swiping to the second or third slide, then swiping down causes the carousel to go back to the first slide. A page with amp-carousel 0.2 does not see the same issue:

https://output.jsbin.com/hudalit/1

**Comment by itsmsc** [comment]
You can see the problem on one of our pages: https://www.mdr.de/tv/index~amp.html
Both problems, described on the first entry; _If "autoplay" is set on the carousel then the carousel will be animating indefinitely while scrolling. If "autoplay" is not set, then the carousel will animate back to the first card and stay there._
Please fix this contemporary!

**Comment by joshcp** [comment]
Two additional examples:

- https://amp.pga.com/events/pgachampionship/2019 (Carousel is near top of page)
- https://amp.zales.com/ (Carousel is near bottom of page)

**Comment by itsmsc** [comment]
>  A page with amp-carousel 0.2 does not see the same issue:
> 
> https://output.jsbin.com/hudalit/1

I can confirm that amp-carousel 0.2 does not see the same issue. I have migrate all pages to v 0.2 with some CSS - changes. This works for me. Thank you.

**Comment by tanakayuki** [comment]
I can confirm that the same issue has been fixed by changing to amp-carousel 0.2 
(iOS 13 beta in browserstack)

Thank you!!

**Comment by conwaydev** [comment]
Noticed this issue this morning with `type="slides"` and `autoplay`.

**Comment by joshcp** [comment]
Thanks @sparhami - will this fix go live Tuesday? Or, what's the timeline?

**Comment by sparhami** [comment]
> Thanks @sparhami - will this fix go live Tuesday? Or, what's the timeline?

The normal release process would be that this will be in the next canary on Tuesday, which will go live one week after that. I will look into cherry-picking this into production however, since this does seem like it causes quite a few problems. Hopefully that can be done today or Monday.

**Comment by 7110** [comment]
@sparhami 
I think this `amp-carousel` bug https://github.com/ampproject/amphtml/issues/24844 which  is not been cured yet by https://github.com/ampproject/amphtml/pull/24771 .
`amp-carousel` will go next even when scrolling in the X-axis direction.

http://hml.vivo4g.clientes.ananke.com.br/videos/erro-carousel-iphone2.webm

iOS 13.1.2 Safari (iPhone7, iPhoneX, etc...)


Please check this bug 🙇🙇🙇

**Comment by 7110** [comment]
@sparhami 
I think this `amp-carousel` bug https://github.com/ampproject/amphtml/issues/24844 which  is not been cured yet by https://github.com/ampproject/amphtml/pull/24771 .
`amp-carousel` will go next even when scrolling in the X-axis direction.

http://hml.vivo4g.clientes.ananke.com.br/videos/erro-carousel-iphone2.webm

iOS 13.1.2 Safari (iPhone7, iPhoneX, etc...)
(used `dist/v0/amp-carousel-0.1.js` which pulled the latest master branch and generated by `gulp dist`)

Please check this bug 🙇🙇🙇

**Comment by ThongSaltire** [comment]
Confirmed! This bug is still existed with current amp-carousel version

**Comment by sparhami** [comment]
While the bug has been fixed in the code, it is not live yet. The fix will hopefully be present in the next canary version. The [releases Slack channel](https://app.slack.com/client/T0ADHJGD6/C4NVAR0H3) is updated whenever a new canary is cut and when a canary is promoted to production.

**Comment by acsant** [comment]
![SwipeCarouselBug](https://user-images.githubusercontent.com/5479907/66856525-410ca480-ef3a-11e9-8e7c-5958bb8b8ec4.gif)

Should this issue be re-opened? I'm noticing still noticing this bug, but only when scroll after swiping out of the full screen in the new carousel v0.2

**Comment by sparhami** [comment]
@acsant  To keep the issues separate and easy to follow, please file a new bug for amp-carousel 0.2.

**Comment by Aeakanshi** [comment]
We have also faced the same scrolling issue in ios devices.

**Comment by Aeakanshi** [comment]
For the Scroll function use this CSS code, it works

{
overflow-y: scroll; 
  -webkit-overflow-scrolling: touch;
}
